src/lug/Window/Win32/WindowImplWin32.hpp

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include <lug/Window/Window.hpp>
+
+namespace lug {
+namespace Window {
+namespace priv {
+class WindowImpl {
```

Comment:
Where are the move constructor and the move assignment operator? (`= delete` I think)